CMakeLists.txt

```diff
@@ -1,13 +1,15 @@
 cmake_minimum_required(VERSION 3.0.0)
-project("Lily")
+project("Lily" C)
 
 if(MSVC)
     add_definitions(/W4)
     add_definitions(/wd4201)
     add_definitions(/wd4204)
     add_definitions(/wd4214)
     add_definitions(/MP)
+    set(CMAKE_CTEST_COMMAND	copy lily.exe ..)
 else()
+    set(CMAKE_CTEST_COMMAND	cp lily ..)
```

Comment:
Please change this. My lily directory is called lily, so this call to `cp` fails.